api/src/main/java/org/openmrs/api/OrderSetService.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api;
+
+import org.openmrs.OrderSet;
+import org.openmrs.annotation.Authorized;
+import org.openmrs.api.db.OrderSetDAO;
+import org.openmrs.util.PrivilegeConstants;
+
+import java.util.List;
+
+/**
+ * @since 1.12
+ * Contains methods pertaining to creating/deleting/voiding Order Sets
+ */
+public interface OrderSetService extends OpenmrsService {
+	
+	/**
+	 * Setter for the OrderSet data access object. The dao is used for saving and getting orders
+	 * to/from the database
+	 *
+	 * @param dao The data access object to use
+	 */
+	void setOrderSetDAO(OrderSetDAO dao);
+	
+	/**
+	 * Save or update the given <code>orderSet</code> in the database. If the OrderSet is retired
+	 * it will set retired by and retired date.
```

Comment:
Is this comment supposed to be here? Because this should only happen when one calls retire/unretire methods below